pkg/controller/import-controller.go

```diff
@@ -705,8 +705,8 @@ func getSource(pvc *corev1.PersistentVolumeClaim) string {
 	return source
 }
 
-// returns the source string which determines the type of source. If no source or invalid source found, default to http
-func getContentType(pvc *corev1.PersistentVolumeClaim) string {
+// GetContentType returns the content type of the source image. If invalid or not set, default to kubevirt
+func GetContentType(pvc *corev1.PersistentVolumeClaim) string {
 	contentType, found := pvc.Annotations[AnnContentType]
 	if !found {
 		contentType = ""
```

Comment:
I think we should just return cdiv1.DataVolumeKubeVirt if the annotation does not exist, that is the default value if omitted.

Reply:
Sure, that's what the function returns in this case, but I agree an immediate return is better. Fixed.